libsession/src/main/java/org/session/libsession/messaging/sending_receiving/ReceivedMessageHandler.kt

```diff
@@ -159,14 +168,13 @@ fun MessageReceiver.cancelTypingIndicatorsIfNeeded(senderPublicKey: String) {
 private fun MessageReceiver.handleExpirationTimerUpdate(message: ExpirationTimerUpdate) {
     SSKEnvironment.shared.messageExpirationManager.insertExpirationTimerMessage(message)
 
-    // TODO (Groups V2 - FIXME)
-    val isGroupV1 = message.groupPublicKey != null
+    val isGroupV1 = message.groupPublicKey != null && message.groupPublicKey?.startsWith(IdPrefix.GROUP.value) == false
```

Comment:
We should stick with our  naming, so it looks like this should be isLegacyGroup?

Reply:
Done